src/base/bittorrent/trackerentry.cpp

```diff
@@ -50,35 +55,59 @@ QString TrackerEntry::url() const
     return QString::fromStdString(nativeEntry().url);
 }
 
+QStringList TrackerEntry::messages() const
+{
+    // Use QStringList as opposed to QSet. In practice there will be only
+    // few messages stored and QSet would not be optimal for our use case.
+    QStringList messages;
+    const auto addMessage = [&messages](const std::string &msg) {
```

Comment:
Coding style is broken. You should move `{` to the next line.
And please check all the similar code.